website/docs/swizzling.md

```diff
@@ -207,12 +207,16 @@ Wrapping a theme is a great way to **add extra components around existing one**
 import React from 'react';
 import BlogPostItem from '@theme-original/BlogPostItem';
 import MyCustomCommentSystem from '@site/src/MyCustomCommentSystem';
+import { useBlogPost } from '@docusaurus/theme-common/internal':
```

Comment:
We don't document internal hooks in docs, so waiting for @slorber to figure out a better way to handle this.

Reply:
ok, I think maybe it is the most simple way.